vardict.pl

```diff
@@ -475,13 +475,15 @@ sub somdict {
 	 my $vartype = "";
 	 if( ! $vars1->{ $p } ) { # no coverage for sample 1
 	     foreach my $vref2 (@{$vars2->{$p}->{ VAR }}) {
+		next if ($vref2->{ refallele } eq $vref2->{ varallele });
```

Comment:
The algorithm in `findconseq()` causes problems with REF=ALT (issue 181 in Java), but as a quick solution, we can skip variations where refallele==varallele here before output.